core-tests/src/test/java/org/mule/runtime/core/internal/processor/interceptor/ReactiveInterceptorAdapterTestCase.java

```diff
@@ -205,6 +206,7 @@ public void interceptorApplied() throws Exception {
 
       assertThat(((InternalEvent) result).getInternalParameters().entrySet(), hasSize(0));
       verifyParametersResolvedAndDisposed(times(1));
+      verify(interceptor, atLeastOnce()).getClass().getClassLoader();
```

Comment:
i think this is just testing that getClass was called, but not getClassloader. Is there a more expressive way to verify the expected behavior?